dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlanParser.cs

```diff
@@ -135,23 +163,35 @@ internal static Plan ToPlanFromXml(this string xmlString, string goal, SKContext
                         }
                         else
                         {
-                            context.Log.LogTrace("{0}: appending function node {1}", parentNodeName, skillFunctionName);
-                            plan.AddSteps(new Plan(childNode.InnerText));
+                            if (allowMissingFunctions)
+                            {
+                                context.Log.LogTrace("{0}: allowing missing function node {1}", parentNodeName, skillFunctionName);
+                                plan.AddSteps(new Plan(childNode.InnerText));
```

Comment:
from CC testing, the InnerText usually is empty so there's no telling what function this was unless you look at the XML. Can you use childNode.name or skillFunctionName here instead?

Reply:
Good call. Updating.